### PR TITLE
Hide ticket number from resolve API.

### PR DESCRIPTION
**Pull request by LautaroCesso**

### CODE REVIEW

File: client/src/app/main/dashboard/dashboard-create-ticket/create-ticket-form.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ class CreateTicketForm extends React.Component {
     };
 
     state = {
+        display: false,
```
Comment (ivandiazwm) [inline client/src/app/main/dashboard/dashboard-create-ticket/create-ticket-form.js:32]: shouldn't this be `true` by default? Also maybe change name to `formDisplayed`

File: client/src/app/main/dashboard/dashboard-create-ticket/create-ticket-form.scss [hunk 2]
```diff
@@ -1,5 +1,13 @@
 .create-ticket-form {
 
+    &__hide {
+        display: none;
+    }
+
+    &__show {
```
Comment (ivandiazwm) [inline client/src/app/main/dashboard/dashboard-create-ticket/create-ticket-form.scss:7]: not requried class

File: client/src/app/main/dashboard/dashboard-create-ticket/dashboard-create-ticket-page.js [hunk 3]
```diff
@@ -33,10 +33,8 @@ class DashboardCreateTicketPage extends React.Component {
     }
 
     onCreateTicketSuccess(result, email, message) {
-        if((this.props.location.pathname !== '/create-ticket')) {
+        if(this.props.location.pathname !== '/create-ticket') {
             history.push(`/dashboard?message=${message}`);
-        } else {
-            setTimeout(() => {history.push('/check-ticket/' + result.data.ticketNumber + '/' + email)}, 1000);
         }
```
Comment (ivandiazwm) [inline client/src/app/main/dashboard/dashboard-create-ticket/dashboard-create-ticket-page.js:38]: does the message shows? Can you add a screenshot of how it looks?

File: client/src/app/main/dashboard/dashboard-create-ticket/create-ticket-form.js (hunk 1 above)
Comment (LautaroCesso) [reply client/src/app/main/dashboard/dashboard-create-ticket/create-ticket-form.js:32]: okey

File: client/src/app/main/dashboard/dashboard-create-ticket/create-ticket-form.scss (hunk 2 above)
Comment (LautaroCesso) [reply client/src/app/main/dashboard/dashboard-create-ticket/create-ticket-form.scss:7]: okey

File: client/src/app/main/dashboard/dashboard-create-ticket/dashboard-create-ticket-page.js (hunk 3 above)
Comment (LautaroCesso) [reply client/src/app/main/dashboard/dashboard-create-ticket/dashboard-create-ticket-page.js:38]: Screenshot from 2020-05-06 14-14-13

**Pull request closed without merging**